react-native-purchases-ui/package.json

```diff
@@ -0,0 +1,114 @@
+{
+  "name": "react-native-purchases-ui",
+  "title": "React Native Purchases UI",
+  "version": "7.4.0-SNAPSHOT",
```

Comment:
Hmm I see we have this version in the `paywalls` branch, but I wonder if this should be a beta version... I guess we didn't merge the release branch back here... So I'm ok with this for now

Reply:
I also noticed that, I am not sure why `paywalls` doesn't have the version pointing to a beta